test/MatrixProductOperator_test.jl

```diff
@@ -54,6 +54,20 @@
                 arrays = [rand(1, 1, 2, 2), rand(1, 1, 2, 2), rand(1, 1, 2, 2)]
                 MatrixProductOperator{Open}(arrays) isa TensorNetwork{MatrixProductOperator{Open}}
             end
+
+            @testset "`Tensor.meta[:alias]`" begin
```

Comment:
Create a `@testset "Metadata"` that groups this test set.
```suggestion
            @testset "alias" begin
```